main/src/main/java/cgeo/geocaching/connector/gc/AbstractListAdapter.java

```diff
@@ -30,6 +42,92 @@ protected static final class ViewHolder extends AbstractRecyclerViewHolder {
         }
     }
 
+
+    private static class TouchListener implements View.OnClickListener, View.OnTouchListener {
```

Comment:
it seems odd to have something as "generic" as this deeply buried in the gc connector part of c:geo. But then, the same goes with having an "AbstractListActivity" and an "AbstractListAdapter" specific for GC connector. So that's not something to be discussed as part of this PR.